contrib/kind.sh

```diff
@@ -45,41 +47,44 @@ usage() {
-    echo "                                   github CI to be updated with IPv6 settings."
-    echo "                                   DEFAULT: Don't allow."
-    echo "-gm  | --gateway-mode              Enable 'shared' or 'local' gateway mode."
-    echo "                                   DEFAULT: shared."
-    echo "-ov  | --ovn-image            	   Use the specified docker image instead of building locally. DEFAULT: local build."
-    echo "-ml  | --master-loglevel           Log level for ovnkube (master), DEFAULT: 5."
-    echo "-nl  | --node-loglevel             Log level for ovnkube (node), DEFAULT: 5"
-    echo "-dbl | --dbchecker-loglevel        Log level for ovn-dbchecker (ovnkube-db), DEFAULT: 5."
-    echo "-ndl | --ovn-loglevel-northd       Log config for ovn northd, DEFAULT: '-vconsole:info -vfile:info'."
-    echo "-nbl | --ovn-loglevel-nb           Log config for northbound DB DEFAULT: '-vconsole:info -vfile:info'."
-    echo "-sbl | --ovn-loglevel-sb           Log config for southboudn DB DEFAULT: '-vconsole:info -vfile:info'."
-    echo "-cl  | --ovn-loglevel-controller   Log config for ovn-controller DEFAULT: '-vconsole:info'."
-    echo "-dl  | --ovn-loglevel-nbctld       Log config for nbctl daemon DEFAULT: '-vconsole:info'."
-    echo "-ep  | --experimental-provider     Use an experimental OCI provider such as podman, instead of docker. DEFAULT: Disabled."
-    echo "-eb  | --egress-gw-separate-bridge The external gateway traffic uses a separate bridge."
-    echo "--delete                      	   Delete current cluster"
+    echo "-cf  | --config-file                Name of the KIND J2 configuration file."
+    echo "                                    DEFAULT: ./kind.yaml.j2"
+    echo "-kt  | --keep-taint                 Do not remove taint components."
+    echo "                                    DEFAULT: Remove taint components."
+    echo "-ha  | --ha-enabled                 Enable high availability. DEFAULT: HA Disabled."
+    echo "-ho  | --hybrid-enabled             Enable hybrid overlay. DEFAULT: Disabled."
+    echo "-ds  | --disable-snat-multiple-gws  Disable SNAT for multiple gws. DEFAULT: Disabled."
+    echo "-dp  | --disable-pkt-mtu-check      Disable checking packet size greater than MTU. Default: Disabled"
+    echo "-nf  | --netflow-targets            Comma delimited list of ip:port netflow collectors. DEFAULT: Disabled."
+    echo "-sf  | --sflow-targets              Comma delimited list of ip:port sflow collectors. DEFAULT: Disabled."
+    echo "-if  | --ipfix-targets              Comma delimited list of ip:port ipfix collectors. DEFAULT: Disabled."
+    echo "-ifs | --ipfix-sampling             Rate at which packets should be sampled and sent to each target collector. DEFAULT: 400."
+    echo "-ifm | --ipfix-cache-max-flows      Maximum number of IPFIX flow records that can be cached at a time. If 0, caching is disabled. DEFAULT: Disabled."
```

Comment:
is it the current behaviour, cache being disabled?

Reply:
A quick look at the OVS code seems to indicate that if any of those parameters is 0, caching is disabled (behavior is not 100% equal but close enough)